app/api/entities/project_analysis.rb

```diff
@@ -2,6 +2,22 @@ class Entities::ProjectAnalysis < Entities::BaseEntity
   expose :velocity
   expose :volatility
   expose :current_iteration_number
+  expose :date_for_iteration
```

Comment:
This method needs a better name. Perhaps, `next_iteration_date` describes better what the method is doing.